core/src/sumeragi/message.rs

```diff
@@ -76,6 +76,22 @@ pub enum Message {
     ViewChangeSuggested,
 }
```

Comment:
```suggestion
}
```

we should not need this variant in data messages anymore